app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Shopping List</h1>
+
+<h2>Shopping lists</h2>
+<table>
```

Comment:
For the shopping list, the project requirements state that you should count the `total number of food items` and the `total price of the missing food`.

![image](https://github.com/Fene-87/recipe_app/assets/38088818/f16d8297-917d-4139-a7eb-67544ee454b7)

![image](https://github.com/Fene-87/recipe_app/assets/38088818/3413cf80-9446-44d9-8471-96c4b0c5e94c)

Your Shopping list:

![image](https://github.com/Fene-87/recipe_app/assets/38088818/2f60a9cf-7d08-4d02-9df5-1d1e1eb1e83e)

- It would be best to also `center` your heading and shopping list table to match the layout of the wireframe.

Also, the shopping list should show the list of food that is missing for all recipes of the logged-in user (compare the list of food for all recipes with the general food list of that user).

![image](https://github.com/Fene-87/recipe_app/assets/38088818/f14b8713-3777-41ea-aa8f-ecd60d0df4fe)

- This one is a bit tricky, note this means that the shopping list should only display ingredients that are missing from inventory. 
For example, Let's say the quantity of tomatoes added to the food list is 2. On the recipe details, I add 5 tomatoes required for the recipe as an ingredient. That means I'm short of 3 tomatoes that I do not have in the inventory. Therefore, 3 tomatoes should be added to the shopping list.

Currently, your shopping list displays all ingredients from a recipe even if there's enough quantity in the food list per item. I hope this makes sense.